### PR TITLE
Added ability to edit the payload on the queues

### PR DESCRIPTION
**Pull request by PatrickWaweru**
- Added ability to edit the payload on the queues and re-queue
- Fixed CSS, Validation, Re-queue

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/afyastat/api/db/hibernate/HibernateMedicOutgoingRegistrationDao.java [hunk 1]
```diff
@@ -228,8 +228,39 @@ public MedicOutgoingRegistration getRecordByPatientAndPurpose(Integer ptId, Stri
...
+	 * 
+	 * @param uuid the uuid of the record
+	 * @param status the status to set
+	 */
+	@Override
+	public MedicOutgoingRegistration recordSetStatus(String uuid, Integer status) {
+		Criteria criteria = getSession().createCriteria(MedicOutgoingRegistration.class);
+		criteria.add(Restrictions.eq("uuid", uuid));
+		MedicOutgoingRegistration record = (MedicOutgoingRegistration) criteria.uniqueResult();
+		record.setStatus(status);
```
Comment (ojwanganto) [inline api/src/main/java/org/openmrs/module/afyastat/api/db/hibernate/HibernateMedicOutgoingRegistrationDao.java:248]: is the intention to set status and save, or just set and leave?

File: api/src/main/java/org/openmrs/module/afyastat/api/db/hibernate/HibernateMedicOutgoingRegistrationDao.java [hunk 2]
```diff
@@ -228,8 +228,39 @@ public MedicOutgoingRegistration getRecordByPatientAndPurpose(Integer ptId, Stri
...
+	 * 
+	 * @param uuid the record uuid
+	 * @param payload the record payload
+	 */
+	@Override
+	public MedicOutgoingRegistration recordSetPayload(String uuid, String payload) {
+		Criteria criteria = getSession().createCriteria(MedicOutgoingRegistration.class);
+		criteria.add(Restrictions.eq("uuid", uuid));
+		MedicOutgoingRegistration record = (MedicOutgoingRegistration) criteria.uniqueResult();
+		record.setPayload(payload);
```
Comment (ojwanganto) [inline api/src/main/java/org/openmrs/module/afyastat/api/db/hibernate/HibernateMedicOutgoingRegistrationDao.java:263]: same as the comment above

**Review by ojwanganto — CHANGES_REQUESTED** [review]

File: api/src/main/java/org/openmrs/module/afyastat/api/db/hibernate/HibernateMedicOutgoingRegistrationDao.java (hunk 2 above)
Comment (PatrickWaweru) [reply api/src/main/java/org/openmrs/module/afyastat/api/db/hibernate/HibernateMedicOutgoingRegistrationDao.java:263]: To set and save. @Transaction saves it automatically

Comment (ojwanganto) [reply api/src/main/java/org/openmrs/module/afyastat/api/db/hibernate/HibernateMedicOutgoingRegistrationDao.java:263]: thanks. sorry

**Pull request merged**